sys/tests/vfs.c

```diff
@@ -54,35 +54,6 @@ static int test_vfs(void) {
   vnode_drop(dev_zero);
   assert(dev_zero->v_usecnt == 1);
 
-  uio_t uio;
```

Comment:
I'd expect you to rewrite following tests instead of just removing the code.

Reply:
Ok. I verified that `/dev/zero` and `/dev/null` are throughly tested by `utest/fd.c`. OTOH testing `/dev/cons` has little use.